db/migrate/20230214083343_add_signed_up_for_adviser_to_application_forms.rb

```diff
@@ -0,0 +1,5 @@
+class AddSignedUpForAdviserToApplicationForms < ActiveRecord::Migration[7.0]
+  def change
+    add_column :application_forms, :signed_up_for_adviser, :boolean, default: false
```

Comment:
i wonder if it would be more useful to have a `signed_up_for_adviser` nullable date-time field, where `NULL` means "not signed up"? from an analytics point of view, it might be useful to have the date.

Reply:
I thought about this as well but decided against it; the flag indicates that someone has signed up for an adviser but we only know when they signed up for an adviser if they did it through Apply as the API just returns a boolean when we check if an existing candidate has an adviser. In this scenario the time would represent when we checked and not necessarily when they signed up.

Reply:
ok thanks, makes sense